functests/utils/profilesupdate/profile_update.go

```diff
@@ -0,0 +1,54 @@
+package profilesupdate
+
+import (
+	corev1 "k8s.io/api/core/v1"
+
+	mcv1 "github.com/openshift/machine-config-operator/pkg/apis/machineconfiguration.openshift.io/v1"
+
+	performancev2 "github.com/openshift-kni/performance-addon-operators/api/v2"
+	testutils "github.com/openshift-kni/performance-addon-operators/functests/utils"
+	testlog "github.com/openshift-kni/performance-addon-operators/functests/utils/log"
+	"github.com/openshift-kni/performance-addon-operators/functests/utils/mcps"
+	"github.com/openshift-kni/performance-addon-operators/functests/utils/profiles"
+	"github.com/openshift-kni/performance-addon-operators/pkg/controller/performanceprofile/components"
+	profilecontroller "github.com/openshift-kni/performance-addon-operators/pkg/controller/performanceprofile/components/profile"
+)
+
+func UpdateIsolatedReservedCpus(isolatedSet performancev2.CPUSet, reservedSet performancev2.CPUSet) {
+	profile, err := profiles.GetByNodeLabels(testutils.NodeSelectorLabels)
+	if err != nil {
+		testlog.Errorf("could not get the performance profile: %v", err)
+	}
+	testlog.Infof("updating performance profile %q with new isolated cpus set: %q and new reserved cpus set: %q", profile.Name, isolatedSet, reservedSet)
+
+	updatedProfile := profile.DeepCopy()
+	updatedProfile.Spec.CPU = &performancev2.CPU{
+		Isolated: &isolatedSet,
+		Reserved: &reservedSet,
+	}
+
+	ApplyProfile(updatedProfile)
+}
+
+func ApplyProfile(profile *performancev2.PerformanceProfile) {
```

Comment:
I would expect this function to return an error status whether the update finished successfully.